examples/fleetpub.ts

```diff
@@ -0,0 +1,52 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import RclnodejsTransport from '@osrf/romi-js-rclnodejs-transport';
+
+//example of how one could publish fleet state to the  web
+
+async function pub(): Promise<void> {
+    const transport = await RclnodejsTransport.create("example");
+
+    const pub= transport.createPublisher(RomiCore.fleetStates);
+    let list=[4,5,6];
+    let msg:RomiCore.FleetState = {
+        name:           "WebRobofleet",
+        robots: []
+    }
+    for (let i in list){
+        msg.robots.push(retRobo(i));
+    }
+    var someTimer = setInterval(()=>{
```

Comment:
please do not use var, use const if possible or let for mutable data. Also `someTimer` is not used.